docs/changelog-fragments.d/459.feature.md

```diff
@@ -0,0 +1,9 @@
+Added the ability to run builder commands to create execution
+environment using mode `stdout`.
+
+Here is an example invocation:
+```bash
+ansible-navigator builder build --tag ee_image_name --workdir ~/manifest-file --container-runtime podman -v 3 --mode stdout
```

Comment:
(question, optional) I wonder if it would be better from the readability/rendering perspective to provide multiline commands in code snippets. This would be useful for people with larger fonts (i.e. less letters per line), narrow browser windows, mobile screens.
```suggestion
ansible-navigator builder \
  build \
  --tag ee_image_name \
  --workdir ~/manifest-file \
  --container-runtime podman \
  -v 3 \
  --mode stdout
```

Reply:
I find multi-line comment is difficult to read especially on a monitor with a medium font. Hence I will keep the format as it is.